app/controllers/api/attempts/events_controller.rb

```diff
@@ -13,7 +13,11 @@ class EventsController < ApplicationController
       before_action :authenticate_client, only: :poll
 
       def poll
-        head :method_not_allowed
+        redis_client.delete_events(
```

Comment:
I think we may want to skip calling this unless there are acks provided

Reply:
yeah i think that's right! updated in https://github.com/18F/identity-idp/pull/11995/commits/f797550947585ff9ed81e83331db57d280c7cd53 and i added a couple more tests